src/shared/pages/list/mpamBusinessUnits/amendBusinessUnit.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { Reducer, useEffect } from 'react';
```

Comment:
File name is amendBusinessUnit but inside the file it's businessArea, is this intentional?

Reply:
No it should all be about amending units, I'll change it